docs/scos/dev/feature-integration-guides/202204.0/glue-api/glue-backend-api/glue-api-authorization-protected-endpoints-integration.md

```diff
@@ -0,0 +1,204 @@
+| AuthorizationContextExpanderPlugin | Adds the information about protected paths to the documentation generation context.  | Spryker\\Glue\\GlueStorefrontApiApplicationAuthorizationConnector\\Plugin\\DocumentationGeneratorApi |
+| ProtectedPathAuthorizationStrategyPlugin | Authorization rule for the route that uses the current strategy.                     | Spryker\\Client\\GlueStorefrontApiApplicationAuthorizationConnector\\Plugin\\Authorization                                         |
+
+**Backend API plugins**
+
+| PLUGIN | SPECIFICATION | NAMESPACE                                                                                         |
+| --- | --- |---------------------------------------------------------------------------------------------------|
+| IsProtectedTableColumnExpanderPlugin | Provides capability to extends route description table. | Spryker\\Glue\\GlueBackendApiApplicationAuthorizationConnector\\Plugin\\GlueApplication           |
+| AuthorizationContextExpanderPlugin | Adds the information about protected paths to the documentation generation context. | Spryker\\Glue\\GlueBackendApiApplicationAuthorizationConnector\\Plugin\\DocumentationGeneratorApi |
+| ProtectedPathAuthorizationStrategyPlugin | Authorization rule for the route that uses the current strategy. | Spryker\\Zed\\GlueBackendApiApplicationAuthorizationConnector\\Communication\\Plugin\\Authorization                                            |
+
+**src/Pyz/Client/Authorization/****AuthorizationDependencyProvider****.php**
+
+```php
+<?php
+
+namespace Pyz\Client\Authorization;
+
+use Spryker\Client\GlueStorefrontApiApplicationAuthorizationConnector\Plugin\Authorization\ProtectedPathAuthorizationStrategyPlugin;
+
+class AuthorizationDependencyProvider extends SprykerAuthorizationDependencyProvider
+{
+    /**
+     * @return array<\Spryker\Shared\AuthorizationExtension\Dependency\Plugin\AuthorizationStrategyPluginInterface>
+     */
+    protected function getAuthorizationStrategyPlugins(): array
+    {
+        return [
+            new ProtectedPathAuthorizationStrategyPlugin(),
+        ];
+    }
+}
+```
+
+**src/Pyz/Zed/Authorization/****AuthorizationDependencyProvider****.php**
+
+```php
+<?php
+
+namespace Pyz\Zed\Authorization;
+
+use Spryker\Zed\GlueBackendApiApplicationAuthorizationConnector\Communication\Plugin\Authorization\ProtectedPathAuthorizationStrategyPlugin;
+
+class AuthorizationDependencyProvider extends SprykerAuthorizationDependencyProvider
+{
+    /**
+     * @return array<\Spryker\Shared\AuthorizationExtension\Dependency\Plugin\AuthorizationStrategyPluginInterface>
+     */
+    protected function getAuthorizationStrategyPlugins(): array
+    {
+        return [
+            new ProtectedPathAuthorizationStrategyPlugin(),
+        ];
+    }
+}
+```
+
+**src/Pyz/Glue/DocumentationGeneratorApi/****DocumentationGeneratorApiDependencyProvider****.php**
```

Comment:
TBH documentation is not required for auth process. So we can not include it here

Reply:
just delete this one and 2 related plugins AuthorizationContextExpanderPlugin ?

Reply:
Yes. Documentation is related for documentation how-to.